### ISSUE TITLE
Less logging, more timestamps, please

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/treemachine#162]
The treemachine console.log on ot20 (production) was 300+ Mbyte. This seems a bit excessive.

I've put a sample (tail -10000) here:
http://mumble.net/~jar/tmp/console.tail

It would be useful to have timestamps, so we can match Java logging output to apache logs and get a hint at usage patterns.

Thanks

**Comment by jar398** [comment]
data/graph.db/messages.log is also 300M.

**Comment by josephwb** [comment]
From @jar398 on IRC (for my own notes):
____________
08:34 I think I would put: timestamp, arguments, nonce on call; then timestamp, nonce on return  - the nonce being anything that could be used to match call and return, maybe a short unique id (counter) or even the first timestamp then if there’s a call without a return, we’ve nailed it

**Comment by josephwb** [comment]
@jar398 How does this look?
```bash
...
2015-02-10 16:44:39.977  Running subtree service.
	Argument 'ott_id' = 81443
	Argument 'node_id' = 3019459
	Error: Provide only one "node_id" or "ott_id" argument.
2015-02-10 16:44:39.998  Exiting subtree service on error.
2015-02-10 16:44:48.882  Running subtree service.
	Argument 'ott_id' = 81443
2015-02-10 16:44:49.525  Exiting subtree service on success.
2015-02-10 16:45:27.828  Running subtree service.
	Argument 'ott_id' = 814436732
	Error: Invalid "ott_id" argument.
2015-02-10 16:45:27.831  Exiting subtree service on error.
```

**Comment by jar398** [comment]
This looks great - can you summarize what it was doing before that
generated so much output, for comparison? I suspect that only logging
requests will result in a large reduction, but I didn't know what it was
doing before so can't be certain.

**Comment by josephwb** [comment]
Apparently anything that prints to ```System.out``` gets logged in ```"console.log"```. This is stuff we (very verbosely) print to screen when using the command line. Since the plugins use the same code, that was what was causing the huge log files.
